### PR TITLE
feat(Contribution Flow): display minimum amount in StepDetail

### PR DESCRIPTION
**Pull request by snitin315**
Resolve https://github.com/opencollective/opencollective/issues/2008

# Description

display minimum amount in StepDetail
# Screenshots

~~`Screenshot at 2020-06-02 16-05-19`~~

~~`Screenshot at 2020-06-04 15-47-05`~~

Screenshot at 2020-06-29 09-49-06

### CODE REVIEW

File: components/contribution-flow/StepDetails.js [hunk 1]
```diff
@@ -35,6 +37,13 @@ const FrequenciesI18n = defineMessages({
   },
 });
 
+const MinAmount = styled(Span)`
+  &:before {
+    content: 'Minimum Amount: ';
+    margin-left: -90px;
+  }
```
Comment (Betree) [inline components/contribution-flow/StepDetails.js:44]: It's a bit hacky to do it like that, and we need the strings to be internationalized (see [/help/developers/translations](https://docs.opencollective.com/help/developers/translations)).

Let's rather put the message in the DOM, and use flex/margins for positioning.

Comment (snitin315) [reply components/contribution-flow/StepDetails.js]: @Betree Updated :+1: 

Screenshot at 2020-06-04 16-03-10

**Comment by snitin315** [comment]
@Betree I'm not sure what's causing the E2E failure :confused:

**Comment by Betree** [comment]
@snitin315 it's not from you, it's an issue from Stripe that has been fixed in https://github.com/opencollective/opencollective-frontend/pull/4329.

Rebasing your branch on `master` should solve it.

**Comment by snitin315** [comment]
Latest screenshot -

Screenshot at 2020-06-29 09-49-06

**Pull request merged**